spec/views/visitors/index.html.erb_spec.rb

```diff
@@ -45,6 +45,7 @@ def fix_time_at(time)
     end
   end
 
+  let(:invite_text) { 'Want to learn more? Listen in. Next projects review meeting' }
```

Comment:
if we were using l18n we could be looking up this text from locale - certainly for a spec that seems appropriate